packages/vsce/__tests__/__e2e__/wiremock/__files/CICSJVMEndpoint/records.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<response xmlns="http://www.ibm.com/xmlns/prod/CICS/smw2int" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:schemaLocation="http://www.ibm.com/xmlns/prod/CICS/smw2int http://my-localhost:8080/CICSSystemManagement/schema/CICSSystemManagement.xsd" 
+version="3.0" connect_version="0610">
+	<resultsummary api_response1="1024" api_response2="0" api_response1_alt="OK" api_response2_alt="" recordcount="1" displayed_recordcount="1" cachetoken="E1812A8FA4625087" />
```

Comment:
Another point of discussion - all other cachetoken values are REALLY obvious as to what they are (JVMDISABLEDTOKEN, 000PROGRAMSTOKEN). I suggest you do the same as it makes looking through the wiremock logs SOOO much easier to work out where requests are coming from and which ones aren't working etc.

Note: I've kept them all the same length - hence the `0`s at the start